frontend/src/redux/reducers/modal.ts

```diff
@@ -0,0 +1,33 @@
+import {
+  OPEN_SIGNUP,
+  OPEN_LOGIN,
+  CLEAR_MODALS,
+} from '../constants';
+
+const initialState = {
+  signup: false,
+  login: false,
+};
```

Comment:
How about instead of accumulating boolean flags, use a state value that can be checked? Eg 'SIGNUP' or 'LOGIN'

Reply:
yea i reckon that's a good idea, i'll look into that